assets/src/components/nav/topNavMobile.tsx

```diff
@@ -0,0 +1,153 @@
+  closeIcon,
+  hamburgerIcon,
+  logoIcon,
+  questionMarkIcon,
+  refreshIcon,
+  settingsIcon,
+  speechBubbleIcon,
+} from "../../helpers/icon"
+import { toggleMobileMenu, toggleNotificationDrawer } from "../../state"
+import NotificationBellIcon from "../notificationBellIcon"
+import { currentRouteTab } from "../../models/routeTab"
+import { openDrift } from "../../helpers/drift"
+import { reload } from "../../models/browser"
+
+const topNavMobile = (): JSX.Element => {
+  const location = useLocation()
+
+  const [state, dispatch] = useContext(StateDispatchContext)
+
+  const { routeTabs, notificationDrawerIsOpen, mobileMenuIsOpen } = state
+
+  const bellIconClasses = notificationDrawerIsOpen
+    ? ["m-top-nav__notifications-icon", "m-top-nav__notifications-icon--active"]
+    : ["m-top-nav__notifications-icon"]
+
+  let tabName = "Untitled"
+  const showTabName = location.pathname === "/"
+  const currentTab = currentRouteTab(routeTabs)
+  if (showTabName && currentTab) tabName = currentTab.presetName || "Untitled"
+
+  return (
+    <div className="m-top-nav-mobile" data-testid="top-nav-mobile">
+      <div
+        className={
+          "m-top-nav-mobile__menu" +
+          (mobileMenuIsOpen
+            ? " m-top-nav-mobile__menu-open"
+            : " m-top-nav-mobile__menu-closed")
+        }
+      >
+        <div className="m-top-nav-mobile__menu-header">
+          <Link
+            className="m-top-nav__logo"
+            onClick={() => dispatch(toggleMobileMenu())}
+            to="/"
+            title="Skate"
+          >
+            {logoIcon("m-top-nav-mobile__logo-icon")}
+          </Link>
+
+          <button
+            className="m-top-nav-mobile__close"
+            onClick={() => dispatch(toggleMobileMenu())}
+            title="Close"
+          >
+            {closeIcon("m-top-nav-mobile__close-icon")}
+          </button>
+        </div>
+        <ul className="m-top-nav-mobile__links">
+          <li>
+            <button
+              className="m-top-nav-mobile__menu-button"
+              onClick={() => reload()}
+              title="Refresh"
+            >
+              {refreshIcon("m-top-nav-mobile__menu-icon")}
+              Refresh
+            </button>
+          </li>
+
+          <li>
+            <button
+              className="m-top-nav-mobile__menu-button"
+              onClick={openDrift}
+              title="Support"
+            >
+              {speechBubbleIcon("m-top-nav-mobile__menu-icon")}
+              Support
+            </button>
+          </li>
+
+          <li>
+            <button
+              className="m-top-nav-mobile__menu-button"
+              onClick={() => displayHelp(location)}
+              title="About Skate"
+            >
+              {questionMarkIcon("m-top-nav-mobile__menu-icon")}
+              About Skate
+            </button>
+          </li>
+
+          <li>
+            <NavLink
+              className="m-top-nav-mobile__menu-link"
```

Comment:
Perhaps more of a design question, but should this also have a separate active state and corresponding `activeClassName`?

Reply:
I added a matching activeClassName to Left Nav in 29da05c